src/layout/css/style.scss

```diff
@@ -11,4 +11,5 @@
 @import "download";
 @import "about";
 @import "home";
-@import "socialLinks"
+@import "socialLinks";
+@import "Modal"
```

Comment:
Please keep a blank/newline at end of the file.

Reply:
done